### ISSUE TITLE
Explorer files in error state do not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#53140]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.4.3

**Repro Steps:**
1. Launch VS Code
2. Open Explorer(very first button) from the left toolbar.
3. Apply any of the dark theme.

**Actual:**
Color contrast ratio is less than 4.5:1 for some of the contents under Explorer.

**Expected:**
Color Contrast ratio should be equal or greater than 4.5: 1 for each content present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Visually impaired people might not see the texts properly.

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment for Reference: 
![issue3_color contrst_](https://user-images.githubusercontent.com/39204909/41973819-be2ba0d8-7a33-11e8-8b51-1d366dfd6eab.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
This also happens on dark/light themes in the various states (hover/selected)

![image](https://user-images.githubusercontent.com/35271042/42245867-4659bd4c-7ecf-11e8-98f7-866ee5af719a.png)

**Comment by JacksonKearl** [comment]
selected warning files still have 2.1:1 contrast. 

![image](https://user-images.githubusercontent.com/8586769/43612777-283a0c54-9662-11e8-9ce4-9cc4c6c7ede5.png)

**Comment by isidorn** [comment]
@misolori please reopen issues with verification-found

**Comment by misolori** [comment]
Updated the `modified` and `listInactiveSelectionBackground`colors:

| Before  | After |
| ------------- | ------------- |
| <img width="80%" src="https://user-images.githubusercontent.com/35271042/43657877-de6ac834-970b-11e8-94fc-2733984ceb19.png" /><br>Default Contrast Ratio: 3.9:1<br>Selected Contrast Ratio:  2.7:1| <img width="80%" src="https://user-images.githubusercontent.com/35271042/43657887-e61dee12-970b-11e8-8ee2-c00a2f606601.png" /> <br>Default Contrast Ratio: 5.6:1 <br>Selected Contrast Ratio: 4.7:1 |
